parser.y

```diff
@@ -2015,6 +2021,31 @@ SplitOption:
 		}
 	}
 
+SplitSyntaxOption:
+	/* empty */
+	{
+		$$ = &ast.SplitSyntaxOption{}
+	}
+|	"REGION" "FOR"
+	{
+		$$ = &ast.SplitSyntaxOption{
+			HasRegionFor: true,
+		}
+	}
+|	"PARTITION"
```

Comment:
What's the different between `"PARTITION"` and `"REGION" "FOR"`? Could we remove one?

Reply:
`PARTITION` used to indicate splitting partition table.
`region for` will make this syntax more readable.  `split region for table ` is meaningful than `split table`.